apps/roam/src/components/settings/NodeConfig.tsx

```diff
@@ -147,16 +169,114 @@ const NodeConfig = ({
           id="canvas"
           title="Canvas"
           panel={
-            <div className="flex flex-col gap-4 p-1">
+            <div className="flex max-h-[70vh] flex-col gap-4 overflow-y-auto p-1">
               <DiscourseNodeCanvasSettings uid={canvasUid} />
-              <FlagPanel
-                title="Graph Overview"
-                description="Whether to color the node in the graph overview based on canvas color.  This is based on the node's plain title as described by a \`has title\` condition in its specification."
-                order={0}
-                parentUid={node.type}
-                uid={graphOverviewUid}
-                value={node.graphOverview}
-              />
+              <div className="mb-4">
```

Comment:
If we change the from using the node as the setting to `true`/`false` child, we would also need to make it backwards compatible.

As this is out of scope for the PR and the proper fix for the bug you identified would actually be a larger refactor, I'm just going to revert these changes and open a new ticket for the bug. 👍